tools/benchmark_tool/openvino/tools/benchmark/parameters.py

```diff
@@ -152,6 +152,8 @@ def parse_args():
     args.add_argument('-imean', '--input_mean', type=str, required=False, default='',
                       help="Optional. Mean values to be used for the input image per channel.\n Values to be provided in the [R, G, B] format. Can be defined for desired input of the model.\n"
                            "Example: -imean data[255,255,255],info[255,255,255]\n")
+    args.add_argument('-do', '--denormals_optimization', type=str2bool, required=False, default=False, nargs='?', const=True,
```

Comment:
```suggestion
    args.add_argument('-dopt', '--denormals_optimization', type=str2bool, required=False, default=False, nargs='?', const=True,
```
I would call it `dopt` in all places. It is more meaningful in my opinion.

Reply:
@jiwaszki ok, agree. will update!